doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,143 @@
+| `req.asr.mon.04` | Network | The NFVI Network Fabric and Network Operating System **must** provide network operational visibility through alarming and streaming telemetry services for operational management, engineering planning, troubleshooting, and network performance optimisation. |
+
+<p align="center"><b>Table 2-6:</b> Containerised Platform: Assurance Requirements.</p>
+
+<a name="2.3.7"></a>
+### 2.3.7 Security
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.sec.gen.01` | General | The Architecture **must** provide workload isolation. |
+| `req.sec.gen.02` | General | The Architecture **must** support policy based RBAC. |
+| `req.sec.gen.03` | General | The Architecture **must** support a centralised authentication and authorisation mechanism. |
+| `req.sec.gen.04` | General | The Architecture **must** provide Operating System kernel isolation. |
+| `req.sec.gen.05` | General | The Architecture **must** provide comoute resources isolation. |
+| `req.sec.gen.06` | General | The Architecture **must** provide storage resources isolation. |
+| `req.sec.gen.07` | General | The Architecture **must** provide network resources isolation. |
+| `req.sec.zon.01` | Zoning | The Architecture **must** support identity management. |
+| `req.sec.zon.02` | Zoning | The Architecture **must** support password encryption. |
+| `req.sec.zon.03` | Zoning | The Architecture **must** support data, at-rest and in-flight, encryption. |
+| `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. |
+| `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. |
+| `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. |
+| `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with ISO 27001 and adhere to the V1.1 NIST Cybersecurity Framework. |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with ISO27001. |
+| `req.sec.ntw.03` | Networking | The Architecture **must** have the underlay network incorporate encrypted and/or private communications channels to ensure its security. |
+| `req.sec.ntw.04` | Networking | The Architecture **must** configure all of the underlay network components to ensure the complete separation from the overlay customer deployments. |
 
-<p align="center"><b>Table 2-1:</b> Kubernetes General Requirements.</p>
+<p align="center"><b>Table 2-7:</b> Containerised Platform: Security Requirements. </p>
```

Comment:
Suggest this is changed to "Kubernetes Architecture: Security Requirements"

Reply:
Thanks - Change made